docs/core/run-time-config/compilation.md

```diff
@@ -8,10 +8,10 @@ ms.topic: reference
 
 ## Tiered compilation
 
-- Configures whether the just-in-time (JIT) compiler uses [tiered compilation](../whats-new/dotnet-core-3-0.md#tiered-compilation).
+- Configures whether the just-in-time (JIT) compiler uses [tiered compilation](../whats-new/dotnet-core-3-0.md#tiered-compilation). Tiered compilation includes two distinct behaviors: [quick JIT](#quick-jit) and the use of [ReadyToRun](../whats-new/dotnet-core-3-0.md#readytorun-images) (pre-compiled) code.
```

Comment:
> Tiered compilation includes two distinct behaviors

Perhaps:
"Tiered compilation transitions methods through two tiers: the first tier generates code more quickly (quick JIT) or loads pre-compiled code (ReadyToRun), and the second tier generates optimized code in the background (optimizing JIT)."